pkgs/development/tools/mysql-shell/default.nix

```diff
@@ -108,14 +106,10 @@ stdenv.mkDerivation rec {
     "-DWITH_LZ4=system"
     "-DWITH_ZLIB=system"
     "-DWITH_PROTOBUF=${protobuf}"
-    "-DHAVE_V8=1"
-    "-DV8_INCLUDE_DIR=${v8}/include"
-    "-DV8_LIB_DIR=${v8}/lib"
+    "-DHAVE_V8=0" # V8 10.x required.
```

Comment:
Why not add v8 10.X instead of removing the feature?

Reply:
V8 is out of mantainance right now. Even mysql-shell is broken because of it. For me, it's not a trival thing to fix V8 errors or package new version of V8.